app/frontend/package-lock.json

```diff
@@ -7236,8 +7236,7 @@
         "ansi-regex": {
           "version": "2.1.1",
           "bundled": true,
-          "dev": true,
-          "optional": true
+          "dev": true
```

Comment:
This can "usually" be avoided by ensuring you run `npm i` or equivalent with the respective `package.json` twice. Not sure why it behaves like that though...